emissions.md

```diff
@@ -35,50 +47,65 @@ CO<sub>2</sub> emissions from gas flaring are not currently included in GCAM.
 
-We summarize here some general points common to non-CO<sub>2</sub> emissions in GCAM
+* The CEDS inventory do not contain emissions for grasslands, forest fires, deforestation and agricultural waste burning on fields. The data for these categories of emissions were added from the GFED LULC data set(as used in CMIP6). 
+
+* CEDS does not have a breakdown of emissions for road transport by mode. Therefore, GAINS emission factors by transport sectors (passenger, freight) and fuels to supplement the CEDS road emissions and derive emissions by different modes using emissions factors from the [GAINS data set](https://iiasa.ac.at/web/home/research/researchPrograms/air/ECLIPSEv5.html).
+
+* Additional information on fluorinated gases is from Guus Velders.
+
+##### Calibration year differences between CEDS and GCAM
+
+Figures 1 and 2 compare historical emissions from CEDS with the emissions from GCAM after initialization . Figure 1 compares global emissions by species and Figure 2 presents a scatter plot comparing emissions by species and year for each region.  As seen in the figures, emissions translate mostly correctly for all species of gases. 
+
+![Figure 1 : Comparison of global CEDS and GCAM emissions in calibration years](images/1a. Global_comparison_of_NonCO2Global.png)
+
+
+![Figure 2 : Comparison of regional CEDS and GCAM emissions in all calibration years](images/1b. Scatterplot comparing emissionsGlobal.png)
+
+However, there are two reasons for differences. The first is that of deforestation emissions from protected land which get zeroed out in GCAM since protected land is held constant in the calibration years. Also deforestation emissions in the final base year are initialized using deforestation coefficients calculated on the basis of deforestation over a 5 year period (2000 and 2005). This leads to a difference in the total deforestation emissions from the CEDS inventory when compared to the numbers initialized in  GCAM. 
+
+Second, the emissions from resource production are different between CEDS and GCAM in calibration years.This happens because the production of fossil fuels in GCAM is broken down by 'vintages' while the emissions factors are calculated in each model year based on total production in each year. When GCAM calculates the emissions, the emissions are calculated for each vintage using the emission factor from that year (for example, the 1975 vintage production will use the 1975 emission factor and the 2015 vintage will use the 2015 emission factor). Since older vintages will have higher emissions factors, the emissions in GCAM will be higher compared to CEDS inventory emissions.
```

Comment:
We incorporated vintage adjustment for GHG emissions from resource production by assigning a “_future.emission.coefficient_” through a “_ResReadInControl_” xml tag. Theoretically, historical GCAM emissions should match input emissions. Now sure why CEDS input emissions and GCAM historical emissions are still different. I suggest just drop this “Second” paragraph.